aiohttp/client_reqrep.py

```diff
@@ -293,6 +295,10 @@ def __init__(
     def __reset_writer(self, _: object = None) -> None:
         self.__writer = None
 
+    @property
+    def skip_auto_headers(self) -> CIMultiDict[None]:
+        return self._skip_auto_headers or CIMultiDict()
```

Comment:
It would be very unexpected that someone would want to get this value back after passing it in, but it was available so to avoid breaking the API it remains exposed.